Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.7-buster as base
+
+ENV PATH="/root/.poetry/bin:$PATH"
+
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+COPY poetry.lock pyproject.toml ./
+
+RUN poetry config virtualenvs.create false
+
+FROM base as production
+
+RUN poetry install --no-dev
+
+WORKDIR /app
+COPY . /app
```

Comment:
Really minor, but I think you could just `COPY todo_app /app/todo_app` rather than everything